`wasm_bindgen` assumed to exist with `#[island]`

**Describe the bug**
Might be a macro hygiene bug, but `wasm_bindgen` is assumed to be a root module with `#[island]`.

**To Reproduce**
install this template: 
```
cargo leptos new --git https://github.com/leptos-rs/start-axum-workspace
```

And change  `app/src/lib.rs` to
```rust
#[leptos::island]
fn App() -> impl leptos::IntoView {}
```

This would give this error when compiling:
```
error[E0433]: failed to resolve: use of undeclared crate or module `wasm_bindgen`
 --> app\src\lib.rs:1:1
  |
1 | #[leptos::island]
  | ^^^^^^^^^^^^^^^^^ use of undeclared crate or module `wasm_bindgen`
  |
  = note: this error originates in the attribute macro `::leptos::wasm_bindgen::prelude::wasm_bindgen` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0425]: cannot find value `FUNCTION` in this scope
 --> app\src\lib.rs:1:1
  |
1 | #[leptos::island]
  | ^^^^^^^^^^^^^^^^^ not found in this scope
  |
  = note: this error originates in the attribute macro `::leptos::wasm_bindgen::prelude::wasm_bindgen` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0405]: cannot find trait `WasmDescribe` in this scope
 --> app\src\lib.rs:1:1
  |
1 | #[leptos::island]
  | ^^^^^^^^^^^^^^^^^ not found in this scope
  |
  = note: this error originates in the attribute macro `::leptos::wasm_bindgen::prelude::wasm_bindgen` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0599]: no function or associated item named `with_hydration` found for struct `SharedContext` in the current scope
 --> app\src\lib.rs:1:1
  |
1 | #[leptos::island]
  | ^^^^^^^^^^^^^^^^^ function or associated item not found in `SharedContext`
  |
  = note: this error originates in the attribute macro `leptos::island` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0599]: no function or associated item named `register_island` found for struct `SharedContext` in the current scope
 --> app\src\lib.rs:1:1
  |
1 | #[leptos::island]
  | ^^^^^^^^^^^^^^^^^
  | |
  | function or associated item not found in `SharedContext`
  | help: there is an associated function with a similar name: `register_suspense`
  |
  = note: this error originates in the attribute macro `leptos::island` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0425]: cannot find function `inform` in this scope
 --> app\src\lib.rs:1:1
  |
1 | #[leptos::island]
  | ^^^^^^^^^^^^^^^^^ not found in this scope
  |
  = note: this error originates in the attribute macro `::leptos::wasm_bindgen::prelude::wasm_bindgen` (in Nightly builds, run with -Z macro-backtrace for more info)
```

**Expected behavior**
To not have an error.

**Additional context**
This is easily fixable by bringing into scope `wasm_bindgen`, this is probably why this have not been discovered yet as `leptos` reexport it, so if you have `use leptos::*;` the problem is not here, but I'm making some macros that generate islands and don't use `use leptos::*;` so for package that don't explicitly depends on `wasm_bindgen` it produce an error.

--- a/leptos_macro/src/component.rs
+++ b/leptos_macro/src/component.rs
@@ -447,7 +447,7 @@ impl ToTokens for Model {
             };
 
             quote! {
-                #[::leptos::wasm_bindgen::prelude::wasm_bindgen]
+                #[::leptos::wasm_bindgen::prelude::wasm_bindgen(wasm_bindgen = ::leptos::wasm_bindgen)]
                 #[allow(non_snake_case)]
                 pub fn #hydrate_fn_name(el: ::leptos::web_sys::HtmlElement) {
                     if let Some(Ok(key)) = el.dataset().get(::leptos::wasm_bindgen::intern("hkc")).map(|key| std::str::FromStr::from_str(&key)) {

